### PR TITLE
add fallback for upstreams of old endpoints

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/pages/EditPage/EditApiForm.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ const propTypes = {
 
 class ApiForm extends PureComponent {
     state = {
-        upstreams: this.props.initialValues.proxy.upstreams,
+        upstreams: this.props.initialValues.proxy.upstreams || {}, // fallback for old endpoins (they have `upstreams: null`), probably temporary
```
Comment (s4nji) [inline src/modules/pages/EditPage/EditApiForm.js]: `endpoins` » `endpoints`

**Comment by rafaeljesus** [comment]
👍

**Comment by ivanzusko** [comment]
👍

**Comment by rafaeljesus** [comment]
👍

**Pull request merged**